src/libraries/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <Compile Include="System.Text.Json.Typeforwards.netcoreapp.cs" />
+    <Compile Include="System.Text.Json.netcoreapp.cs" />
   </ItemGroup>
```

Comment:
what is the value of NetCoreAppMinimum we support for S.T.JSON? I am asking as DateOnly/TimeOnly introduced in .NET 6.0 and cannot be used before this version. I know .NET 5.0 and 3.1 are almost out of support but I wanted to double check here.

Reply:
Minimum version is .NET 6, so this should be fine.